content/knowledge-base/lifecycle.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Product Lifecycle
+---
+
+## Software List
+
+Upbound provides support and software maintenance to paid customers. Supported software products include:
+
+* [UXP](https://www.upbound.io/product/universal-crossplane)
+* [Official Providers](https://marketplace.upbound.io/providers?tier=official)
+
+## Maintenance and Updates
+
+Upbound supports Generally Available releases for up to 12 months. Eligible code-fixes are provided via a new minor release on top of the latest major release branch for up to 3 releases from the most current major release. All software follows semantic versioning of `X.Y.Z`. A major release is a change in the first or second digit. A minor release is a change in the third digit.
```

Comment:
>A major release is a change in the first or second digit. A minor release is a change in the third digit.

This isn't accurate.

The [formal definition](https://semver.org/) is:
```
MAJOR version when you make incompatible API changes
MINOR version when you add functionality in a backward compatible manner
PATCH version when you make backward compatible bug fixes
```

How we use it today at Upbound (and XP) based on my observation is:

- A major release is a change in the first digit and usually indicates breaking API changes or, in the case of 1.0.0, indicates the first release where the API is considered complete (-enough) and stable.
- A minor release is a change in the second digit and usually indicates new features added, and usually also includes bug fixes that might not have been released yet in a previous patch release.
- A patch release is a change in the third digit and only contains bug fixes, no new features.

We have followed this strictly in XP/UXP, less so to date in the providers.
Due to the previous release cadence and pace of development on the providers we simply defaulted to bumping the minor version. We should start following the semver pattern strictly going forward.